test/test_autograd.py

```diff
@@ -4946,7 +4946,113 @@ def test_autograd_simple_views_python(self):
         self._do_test_autograd_simple_views_python(torch.double)
         self._do_test_autograd_simple_views_python(torch.cdouble)
 
-    def test_autograd_complex_views_python(self):
+    def test_autograd_inplace_views_creation_meta(self):
+        # Tests creation_meta properly handled for inplace views
+
+        class Func(torch.autograd.Function):
+            @staticmethod
+            def forward(ctx, x):
+                return x.view_as(x)
+
+            @staticmethod
+            def backward(ctx, x):
+                return x
+        inplace_custom = Func.apply
```

Comment:
This is not inplace, this is a view right?

Reply:
Whoops